burotel/indico_burotel/plugin.py

```diff
@@ -94,3 +97,8 @@ def _before_request(self):
 
     def _override_templates(self, sender, **kwargs):
         return os.path.join(self.root_path, 'template_overrides')
+
+    def _inject_long_term_attribute(self, sender, data, **kwargs):
+        long_term_room_ids = {room.id for room, value in Room.find_with_attribute('long-term') if value}
```

Comment:
since the value is a string (currently we do not support entering non-string values for attributes :/) I'd check for `value == 'true'`

Reply:
perhaps `lower(value) == 'true'` so it's more flexible?

Reply:
`value.lower() in ('true', '1', 'yes')` maybe?